source/fundamentals/connection/connect.txt

```diff
@@ -95,6 +95,14 @@ verify that the connection is successful:
 .. literalinclude:: /code-snippets/connection/stable-api-connect.js
    :language: javascript
 
+.. note::
+
+   As of v4.7, the {+driver-short+} will automatically call the ``MongoClient.connect()``
+   method when using the client to interact with your MongoDB instance, **except** when
```

Comment:
I: the prerequisite of "using the client to interact with..." isn't clear to me. what does the user have to do to get the driver to call this automatically?